hawk/app/assets/stylesheets/shared/_misc.scss

```diff
@@ -32,3 +32,30 @@
     border-radius: 20px;
   }
 }
+
+.diff_sub {
+  background: #fdd;
+}
+
+.diff_add {
+  background: #dfd;
+}
+
+.diff_chg {
+  background: #ddf;
+}
+
+
+td.diff_header {
+color: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented 2 spaces, but was indented 0 spaces